noxfile.py

```diff
@@ -22,6 +22,7 @@
     "pytest-cov",
     "pytest-subtests",
     "pytest-xdist",
+    "pytest-randomly",
```

Comment:
nit: make pytest-randomly in alphabetical order as below?

Reply:
Done!